plugins/modules/mysql_replication.py

```diff
@@ -32,7 +33,9 @@
       C(stopreplica) (STOP REPLICA),
       C(resetprimary) (RESET MASTER) - supported since community.mysql 0.1.0,
       C(resetreplica) (RESET REPLICA),
-      C(resetreplicaall) (RESET REPLICA ALL).
+      C(resetreplicaall) (RESET REPLICA ALL),
+      C(startgroupreplication) (START GROUP_REPLICATION),
+      C(stopgroupreplication) (STOP GROUP_REPLICATION).
```

Comment:
```suggestion
      C(startgroupreplication) (START GROUP_REPLICATION),
      C(stopgroupreplication) (STOP GROUP_REPLICATION).
```
Should we also add ` - supported since community.mysql 3.9.0 like for a choice a bit above?

Reply:
Yes, but use the next release number: 3.10.0